app/viewmodels/journeys/taskList/CapitalAllowancesTasklist.scala

```diff
@@ -48,14 +50,22 @@ object CapitalAllowancesTasklist {
       case Some(Cash) => s".$Cash"
       case _          => ""
     }
-    val tailoringRow = buildSummaryRow(tailoringHref, messages(s"journeys.$CapitalAllowancesTailoring$isCashHeading"), tailoringStatus).some
+
+    val expensesSelected = conditionPassedForViewableLink(TradingAllowancePage, List(TradingAllowance.DeclareExpenses))
+
+    val tailoringRow = returnRowIfConditionPassed(
+      buildSummaryRow(tailoringHref, messages(s"journeys.$CapitalAllowancesTailoring$isCashHeading"), tailoringStatus),
+      expensesSelected
+    )
 
     val capAllowancesTailoringCompleted = tailoringStatus.isCompleted
 
     val zeroEmissionCarsStatus = getJourneyStatus(CapitalAllowancesZeroEmissionCars)(journeyStatuses)
     val zeroEmissionCarsHref   = getUrl(CapitalAllowancesZeroEmissionCars, zeroEmissionCarsStatus, businessId, taxYear)
     val zecIsTailored =
-      conditionPassedForViewableLink(SelectCapitalAllowancesPage, CapitalAllowances.ZeroEmissionCar) && capAllowancesTailoringCompleted
+      conditionPassedForViewableLink(
```

Comment:
could these all use the same `getAllowanceRow` method like the ones below for consistency?